napalm/pyIOSXR/iosxr.py

```diff
@@ -46,6 +48,8 @@
 from napalm.pyIOSXR.exceptions import InvalidInputError
 from napalm.pyIOSXR.exceptions import InvalidXMLResponse
 
+logger = logging.getLogger("napalm.pyiosxr")
```

Comment:
`logger = logging.getLogger(__name__)` is usually more flexible. Would be fine to take this in as-is, but I think you'll have a couple more changes, so might as well include this one too. :wink: